OpenRA.Mods.Common/Traits/Carryall.cs

```diff
@@ -23,6 +23,10 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Transports actors with the `Carryable` trait.")]
 	public class CarryallInfo : TraitInfo, Requires<BodyOrientationInfo>, Requires<AircraftInfo>
 	{
+		[ActorReference]
+		[Desc("Actor type that is initially spawned into this actor. Requires Carryable: trait on the actor.")]
+		public readonly string InitialActor;
```

Comment:
Our coding convention is to explicitly assign a value, even if its the default. This should also be restricted to actors that have the Carryable trait.

Can you please define this as
```csharp
[ActorReference(typeof(CarryableInfo))]
[Desc("Actor type that is initially spawned into this actor.")]
public readonly string InitialActor = null;
```
to address both issues?